microsoft-edge/webview2/concepts/user-data-folder.md

```diff
@@ -1,48 +1,175 @@
+
+
+<!-- ====================================================================== -->
+## Specifying another location
+
+tbd
+
+
+<!-- ====================================================================== -->
+## Error message strings
+
+<!-- When you do x, -->The following error message strings may be returned:
+
+* "User data folder cannot be created because a file with the same name already exists."
+* "Unable to create user data folder, Access Denied."
+* "Edge runtime unable to start"
+* "Failed to create WebView2 environment"
+<!-- show HRESULT? -->
+
+Add code like the following code, such as try/catch code, to handle the error.
+<!-- Will there be an error dialog, or an error code? -->
+
+There's a new API to query where the user data folder is, so you can more easily delete the user data folder, by getting the calculated value of the user data folder location.
+
+*  Win32: [ICoreWebView2Environment7.get_UserDataFolder](/microsoft-edge/webview2/reference/win32/icorewebview2environment7#get_userdatafolder)
+
+*  WPF: link tbd
+
+*  WinForms: same as the WPF link.<!--which is __-->
+
+*  WinUI 2 & WinUI3: For WinUI 2 & WinUI3, there's no such link; you have to use an environment variable instead: `WEBVIEW2_USER_DATA_FOLDER="\<folder path\>"`.  See [CoreWebView2Environment.UserDataFolder Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.userdatafolder).
+<!-- verbose link, delete?
+/dotnet/api/microsoft.web.webview2.core.corewebview2environment.userdatafolder?view=webview2-dotnet-1.0.1083-prerelease -->
+
+
 <!-- ====================================================================== -->
 ## Best practices
 
 User data folders are created automatically by WebView2.  WebView2 developers control the lifetime of the user data folder.  If your application re-uses user data from application sessions, consider saving the user data folders, otherwise you may delete them.  Consider the following scenarios when deciding how to manage your user data folders:
 
-*   If the same user uses your application repeatedly, and the web content of the application relies on the user's data, save the user data folder.  If multiple users use your application repeatedly, create a new user data folder for each new user, and save the user data folder of each user.
-*   If your application does not have repeat users, create a new user data folder for each user, and delete the previous user data folder.
+*  If the same user uses your application repeatedly, and the web content of the application relies on the user's data, save the user data folder.
+
+*  If multiple users use your application repeatedly, create a new user data folder for each new user, and save the user data folder of each user.
+
+*  If your application doesn't have repeat users, create a new user data folder for each user, and delete the previous user data folder.
 
 
 <!-- ====================================================================== -->
 ## Create user data folders
```

Comment:
I think I would call out, that the WV2 will choose a default location for you and that will satisfy most developers needs.  But if the developer has specific needs then they can specify a different location.  And they will need to ensure that that location will be available to the WV2 application runtime for read / write (needed permissions)